Tasks/CopyFilesV2/copyfiles.ts

```diff
@@ -107,6 +108,12 @@ if (matchedFiles.length > 0) {
                 else { // copy
                     console.log(tl.loc('CopyingTo', file, targetPath));
                     tl.cp(file, targetPath);
+                    if (preserveTimestamp) {
+                        let fileStats: tl.FsStats = tl.stats(file);
```

Comment:
const

Reply:
Is it possible tl.stats blows up?

Reply:
Potentially, it looks like in other areas it's doing:
```
                try {
                    targetStats = tl.stats(targetPath);
                }
                catch (err) {
                    if (err.code != 'ENOENT') {
                        throw err;
                    }
                }
```
Would this be your recommendation? I'm not familiar with why this works this way.

Reply:
I that approach generally makes sense. We don't want to fail the whole task on this. My only concern would be the catch block. I think it probably makes sense to just always warn and never throw. Right now you're throwing if the file isn't found, but not doing anything otherwise.

Reply:
Wrapped everything timestamp related in a try/catch and then console.warn in the catch. Let me know your thoughts.

Reply:
That looks good to me.